### PR TITLE
Update the known issues to mention the UnityUI/TMP issue

### PR DESCRIPTION
**Pull request by wiwei**
Updates our known issue with the latest issue we found in https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8556.

### CODE REVIEW

**Review by keveleigh — APPROVED** [review]

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -185,8 +185,19 @@ Motion Controller simulation is now offered in editor play mode along side the e
...
+
 There is currently a known issue for using the [Oculus XR plugin with when targetting Standalone platforms](https://forum.unity.com/threads/unable-to-start-oculus-xr-plugin.913883/).  Check the Oculus bug tracker/forums/release notes for updates.
 
 The bug is signified with this set of 3 errors:
 
 ![Oculus XR Plugin Error](https://forum.unity.com/attachments/erori-unity-png.644204/)
+
+**UnityUI and TextMeshPro**
+
+There's a known issue for newer version of TextMeshPro (2.1.1+) and Unity 2019.4.x, where the default font size for dropdowns has been increased
```
Comment (vaoliva) [inline Documentation/ReleaseNotes.md:198]: Issue also happens with 1.5.0+. Its independent of Unity version. 
Apart from dropdown font size, the character spacing on second column is also altered as you can see in the image.

Comment (vaoliva) [inline Documentation/ReleaseNotes.md:198]:
```suggestion
There's a known issue for newer versions of TextMeshPro (1.5.0+ or 2.1.1+), where the default font size for dropdowns and bold font character spacing has been altered.
```

**Comment by vaoliva** [comment]
Should we also add a note under https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithTheMRTK.html#run-the-handinteractionexamples-scene-in-the-unity-editor ?

**Pull request merged**